node/index.js

```diff
@@ -688,22 +693,6 @@ TChannelConnection.prototype.runInOp = function runInOp(handler, options, sendRe
     }
 };
 
-TChannelConnection.prototype._writeFrame = function _writeFrame(frame, callback) {
-    var self = this;
-    if (self.closing) {
-        self.logger.warn('write frame after close', {
```

Comment:
Yup and that's okay I think, since close should propagate throughout the pipe chain, eventually unified over handler.

Any ops settling after socket close, the best we can do is just proceed for now... is there any easy way to get the warn back from our current pipeline setup? I'm not too concerned if we can't.